.github/docker-soak-tests/otel-collector/collector-config.yml

```diff
@@ -0,0 +1,66 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+  hostmetrics:
+    collection_interval: ${HOSTMETRICS_COLLECTION_INTERVAL_IN_SECONDS}s
+    scrapers:
+      process:
+        include:
+          match_type: strict
+          names:
+            - ${PROCESS_EXECUTABLE_NAME}
+
+processors:
+  filter:
+    metrics:
+      include:
+        match_type: strict
+        metric_names:
+          - process.cpu.time
+          # - process.disk.io
+          - process.memory.physical_usage
+          - process.memory.virtual_usage
+        resource_attributes:
+          - Key: process.command_line
+            Value: ${PROCESS_COMMAND_LINE}
+
+exporters:
+  logging/debug:
+    loglevel: debug
+  logging/error:
+    loglevel: error
+  awsemf:
+    # output_destination: stdout
+    region: us-west-2
+    namespace: ${LOGS_NAMESPACE}
+    log_group_name: ${LOG_GROUP_NAME}
+    log_stream_name: ${LOG_STREAM_NAME}
+    resource_to_telemetry_conversion:
+      enabled: true
+    dimension_rollup_option: NoDimensionRollup
+    metric_declarations:
+      - dimensions: [[process.command_line]]
+        metric_name_selectors:
+          - process.cpu.time
+          # - process.disk.io
+          - process.memory.physical_usage
+          - process.memory.virtual_usage
+
+service:
+  pipelines:
+    traces:
+      receivers:
+        - otlp
+      exporters:
+        - logging/error
+    metrics:
+      receivers:
+        - otlp
+        - hostmetrics
+      processors:
+        - filter
+      exporters:
+        - awsemf
```

Comment:
Think we should only send hostmetrics to awsemf, not the `otlp` metrics

Reply:
Sounds good I can remove the `- otlp` receiver. Although I think I remember the JavaScript SDK checks to see if it can connect to the collector? But I don't know if it goes so far as to check if the receivers it needs are correctly set up. Either way will remove for now.